src/components/task-plan/builder.cjsx

```diff
@@ -100,7 +130,7 @@ module.exports = React.createClass
             </tr>
           </thead>
           <tbody>
-            { _.map periods, @renderPeriodRow }
+            { _.map CourseStore.get(@props.courseId)?.periods, @renderTaskPlanRow }
```

Comment:
Is the `?` needed or should this be covered by the `<Loadable>` guard (and an error otherwise)?